### ISSUE TITLE
Allow the specification of more than one entitlement to `enable` and `disable`

### ISSUE THREAD

**Issue by OddBloke** [canonical/ubuntu-advantage-client#944]
e.g. `ua disable esm-infra esm-apps`, `ua enable esm-apps livepatch`

**Comment by lucasmoura** [comment]
@blackboxsw To allow this, I will need to update the [require_valid_entitlement_name](https://github.com/canonical/ubuntu-advantage-client/blob/master/uaclient/cli.py#L94).  My idea is to verify all entitlements and in the end it would generate a single error message for all of the entitlements that do not exist.

For example, suppose we are running the following command:
`ua disable entitlement1 entitlement2 --assume-yes true`

And both `entitlement1` and `entitlement2` do not exist, the error message will be:
`Cannot disable 'entitlement1', 'entitlement2'`

This would be similar for the `enable` command.

However, if some of the entitlements are found and some are not, should we inform the user that part of the entitlements were found ?

**Comment by lucasmoura** [comment]
Also, should we partially enable/disable entitlements ?

**Comment by blackboxsw** [comment]
@lucasmoura I think we should try to setup anything we can, collect any failures

> Also, should we partially enable/disable entitlements ? For example, if I run:
> 
> `ua enable ent1 ent2`
> 
> But I am only entitled to ent1, should I enable ent1 and say to the user that ent2 is not entitled for his subscription ?

I think we should try to setup any entitlement we can, collect any failures and emit the consolidated failures after trying our best.

**Comment by blackboxsw** [comment]
> @blackboxsw To allow this, I will need to update the [require_valid_entitlement_name](https://github.com/canonical/ubuntu-advantage-client/blob/master/uaclient/cli.py#L94). My idea is to verify all entitlements and in the end generate a single error message for all of the entitlements that do not exist.
> 
> For example, suppose we are running the following command:
> `ua disable entitlement1 entitlement2 --assume-yes true`
> 

+1   collect those failures in one message if possible

> And both `entitlement1` and `entitlement2` do not exist, the error message will be:
> `Cannot disable 'entitlement1', 'entitlement2'`
> 
> This would be similar for the `enable` command.
> 
> However, if some of the entitlements are found and some are not, should we inform the user that part of the entitlements were found ?

Yes, I think we should install everything we can and provide an aggregated error for anything that was unentitled or not present.

**Comment by lucasmoura** [comment]
@blackboxsw Based on your input, I thought about this format for the commands:

Suppose you ran the command: `ua disable ent1 ent2 ent3`
Where:
* ent1 does not exist
* ent2 is not enabled
* ent3 is enabled

In that case, we would produce the following message:

```
Disabling ent2...
ent2 is not currently enabled
See: sudo ua status

Disabling ent3...
Updating packages lists

Result:
ent1 is not a valid entitlement
ent2 was not enabled
ent3 was successfully disabled
```

By doing that we would now update the [require_valid_entitlement_name](https://github.com/canonical/ubuntu-advantage-client/blob/master/uaclient/cli.py#L94) to not abort the command, but just list which ones are valid entitlements or not.

The ideia will be very similar for the `enable` command

**Comment by lucasmoura** [comment]
Fixed by #1055
